api/schema/aws_account_resource.go

```diff
@@ -52,6 +52,9 @@ var (
 				"name": resource_schema.StringAttribute{
 					Description: "Display name for the AWS account.",
 					Required:    true,
+					Validators: []validator.String{
+						stringvalidator.LengthAtLeast(3),
+					},
 				},
 				"organization_master_account_id": StringResourceAttributeWithMode{
```

Comment:
Add an `organization_id` attribute. We won't use it for now on the server side, but when we get around to supporting grouping accounts by organization we'll already have the data for each account.
```
				"organization_id": StringResourceAttributeWithMode{
					StringAttribute: resource_schema.StringAttribute{
						Description: "ID of the AWS organization.",
						Optional:    true,
						PlanModifiers: []planmodifier.String{
							stringplanmodifier.RequiresReplace(),
						},
					},
					attributeWithMode: attributeWithMode{
						Mode: ImmutableAttributeMode,
					},
				},
```